### PR TITLE
Allow autoscaling non-default machine pool with 0 replicas

### PR DESCRIPTION
**Pull request by zgalor**

### CODE REVIEW

**Comment by zgalor** [comment]
@igoihman @nimrodshn @vkareh PTAL

File: cmd/ocm/edit/machinepool/cmd.go [hunk 1]
```diff
@@ -195,10 +196,10 @@ func run(cmd *cobra.Command, argv []string) error {
 	if args.autoscaling.Enabled {
 		asBuilder := cmv1.NewMachinePoolAutoscaling()
 
-		if args.autoscaling.MinReplicas > 0 {
+		if isMinReplicasSet {
```
Comment (vkareh) [inline cmd/ocm/edit/machinepool/cmd.go:199]: You should probably check that `isMinReplicaSet && args.autoscaling.MinReplicas >= 0`

Comment (zgalor) [reply cmd/ocm/edit/machinepool/cmd.go:199]: We don't validate inputs here. We let ocm backend do it and return the error. 
The change was just to allow sending 0 and not treat it as 'not-set'

**Comment by zgalor** [comment]
@vkareh @igoihman @tzvatot this one can already be merged. 

Anyway it depends on the backend to validate input

**Comment by tzvatot** [comment]
LGTM

**Pull request merged**